lib/vendor/coinpayments.js

```diff
@@ -0,0 +1,10 @@
+const Coinpayments = require('coinpayments');
+
+const options = {
+  key: process.env.CP_PUBLIC_KEY,
+  secret: process.env.CP_PRIVATE_KEY
+}
+
+const client = new Coinpayments(options);
```

Comment:
maybe `try`/`catch` this line. If the error message is `'Missing public key and/or secret'` we can just error log it and continue for travis.